tutorials/hello_tupelo.md

```diff
@@ -18,10 +18,82 @@ To the developer it feels like storing it to any datastore such as a
 document database.  After that submit however, the information is being reviewed,
```

Comment:
gossip4 will drop the shell - I think we should still publish this, but noting.

Reply:
So this will need to be removed when we push gossip4.  I guess we will need to decide if there is a "hello world" replacement in WASM or otherwise.